proto/powergate/v1/powergate.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
```

Comment:
New proto file for the main api. This will expand over time as I move definitions out of the old "rpc" module definitions.